core/vibes/soul/sections/slideshow/slideshow.makeswift.tsx

```diff
@@ -0,0 +1,63 @@
+import { Image, Link, List, Number, Shape, Style, TextInput } from '@makeswift/runtime/controls';
+
+import { runtime } from '~/lib/makeswift/runtime';
+
+import { Slideshow } from '.';
+
+interface Slide {
+  title?: string;
+  description?: string;
+  imageSrc?: string;
+  imageAlt?: string;
+  link?: { href?: string; target?: string };
+  label: string;
+}
+
+interface MSAccordionsProps {
+  className: string;
+  slides: Slide[];
+  interval: number;
+}
+
+runtime.registerComponent(
+  function MSSlideshow({ className, slides, interval }: MSAccordionsProps) {
+    return (
+      <Slideshow
+        className={className}
+        interval={interval * 1000}
+        slides={slides.map(({ title, description, imageSrc, imageAlt, link, label }) => {
+          return {
+            title: title ?? '',
+            description: description ?? '',
+            image: { alt: imageAlt ?? '', src: imageSrc ?? '' },
+            cta: { label, href: link?.href ?? '' },
+          };
+        })}
+      />
+    );
+  },
+  {
+    type: 'section-slideshow',
+    label: 'Sections / Slideshow',
+    icon: 'carousel',
+    props: {
+      className: Style(),
+      slides: List({
+        type: Shape({
+          type: {
+            title: TextInput({ label: 'Title', defaultValue: 'Slide Title' }),
+            description: TextInput({ label: 'Description', defaultValue: 'Slide Description' }),
+            imageSrc: Image(),
+            imageAlt: TextInput({ label: 'Image Alt', defaultValue: 'Slide Image' }),
+            link: Link({ label: 'Link' }),
+            label: TextInput({ label: 'CTA Label', defaultValue: 'Learn More' }),
+          },
+        }),
+        getItemLabel(slide) {
+          return slide?.title || 'Slide';
```

Comment:
In the builder, the label is defaulting to `Slide` even though title has the default value of `Slide Title`. Not sure why that default value isn't being picked up initially. 

https://github.com/user-attachments/assets/3803e126-50c3-4831-bbbd-5bf6f7d4cdbd

Reply:
Fixed